R_code/fingerprintplots.r

```diff
@@ -0,0 +1,202 @@
+# plotting GPP vs. R and color coding by season; JAZ & JK; 2018-03-09
+
+#first part is creating dataframes with data we want to plot.
+#starts with reading in individual lake files and binding them into one data frame
+#then merges dataframes together (e.g., metabolism dataframe with loading data frame)
+#there is code in the dropbox for creating a dataframe for the light climate data but I've left this
+#out for now because the light climate data are not on Github
+#code for plotting starts~line 100
+
+
+library(xlsx)
+
+dir<-'C:/Users/jklug/Documents/catchment_metab_wg/results/metab/20161107/' # directory of metabolism data
+folders<-list.files(dir) # folders in this dir
+folders<-folders[-grep('.doc',folders)] # get rid of README doc
+folders<-folders[-grep('Trout',folders)] # skipping trout for now; have to do bootstrapping on this still
+
+all_metab<-data.frame() # data frame to store all metab data
+for(i in 1:length(folders)){ # loops over all folders in metab directory
+  cur<-read.table(file.path(dir,folders[i],paste(folders[i],'_metabEst.txt',sep='')),header=T,sep='\t',
+                  stringsAsFactors = F) # read in lake specific metab data
+  cur<-cur[,1:12] # getting rid of any unnecessary columns
+  cur$lake<-folders[i] #creating a column of lake names from the names of the folders
+  all_metab<-rbind(all_metab,cur) #stacking all the lakes in one dataframe one by one
+}
+
+all_metab$date<-as.Date(paste(all_metab$year,all_metab$doy),format='%Y %j') # making date
+
+# creating season column. season cutoffs used by Jim ; can change if needed
+# Spring = (doy < 180)
+# Summer = (doy >= 180 & doy <=240)
+# Fall = (doy > 240)
+all_metab$season <- rep(NA,nrow(all_metab))
+all_metab$season <- ifelse(all_metab$doy>120&all_metab$doy<180,'spring',all_metab$season) # if doy with criteria, spring, else leave alone
+all_metab$season <- ifelse(all_metab$doy>=180&all_metab$doy<=240,'summer',all_metab$season)
+all_metab$season <- ifelse(all_metab$doy>240&all_metab$doy<305,'fall',all_metab$season)
+
+#loop below creates one dataframe with all loading data in it - JK modifying the loop JZ created above for metabolism
+dir<-'C:/Users/jklug/Documents/catchment_metab_wg/results/nutrient load/' # directory of loading data
```

Comment:
again, don't need full file path 
```suggestion
dir<-'results/nutrient load/' # directory of loading data
```